_static/_init.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the base (and therefore default) configuration to be used by the ATSpectrograph.
+# See tstn-020.lsst.io for a description of how to interact with CSC configurations.
+# The first section of this configuration file contains metadata fields specific to this file.
+# The fields are used for heritage tracking and knowledge transfer only.
+metadata:
+   ticket: DM-24808
+   date_created: 2020-05-24
+   version: v3
+   modifications : Extended maximum linear stage position to support the full range
+   description: >-
+      Contains parameters for ATSpectrograph that are not expected to be modified.
+      Description of each property is found with the schema located in the ts_atspec CSC repository.
+
+# Below are defaults for configurable parameters
```

Comment:
Not all, but between it and the _<site>.yaml file it needs to be fully defined. I've bits of both files

Reply:
Uhm.. but doesn't this means we will have to have some unnecessary site-specific files? for instance, In this case I think the only information that needs to be in the `_summit.yaml` file is the `hostname`, everything else can be in the `_init.yaml` file.

Reply:
I don't think that's true because if we setup testing on the NCSA test stand we won't want those tests to get broken because we changed a filter name or parameter.
It's a bit situation dependent. For this particular CSC I think we need site-dependent files... but it's not always the case.

Reply:
I agree with @tribeiro that I would expect many of our site-specific files to be minimal -- often just a host name. 

In the case of filters can see arguments both ways. But my suggestion is that an integration test that requires some specific information probably would benefit from an override file written for that test. That way the file is available at all locations (including the summit when running the integration test). Then we can either keep the _init.yaml file current with the summit or just keep current data in the _summit.yaml in which case I we'd probably have minimal filter information in _init.yaml.

Reply:
Hmm, ok. I'm not 100% convinced but we'll see how it evolves with time. I fully agree that most site files will only be a hostname.
I've moved everything except the IP address back to the `_init.yaml` file.
I suppose what bothers me is that the option "keep the _init.yaml file current with the summit" is a bit of a slippery slope and somewhat defeats the purpose of a site-specific file in the first place.
@tribeiro - you get the last word on this....

Reply:
I think this the strength of the proposal. We put in `_init.yaml` the baseline for the configuration and in `_<site>.yaml` any customization we need. So, if summit has a `offset_pointing_filter` that is only intended for the summit, it will go to `_summit.yaml`, but `_init.yaml` retain values that can be use elsewhere.